src/BuildScriptGenerator/BaseBashBuildScript.sh.tpl

```diff
@@ -104,7 +106,9 @@ then
 	{{ for excludedDir in DirectoriesToExcludeFromCopyToBuildOutputDir }}
 	excludedDirectories+=" --exclude {{ excludedDir }}"
 	{{ end }}
-	rsync -rtE --links $excludedDirectories . "$DESTINATION_DIR"
+
+	rsync -rEc --links $excludedDirectories . "$DESTINATION_DIR"
```

Comment:
@gpcastro Following are my findings based on which I sent out this PR:
- `--omit-dir-times` alone is not fixing the issue as I am still seeing the `Operation not permitted` errors.
- Removing the `-t` entirely (which includes removing the `omit-dir-times` switch), while does not run into the `Operation not permitted` errors, is worse of all as it is taking lot of time to copy to the destination directory (wwwroot here, which is mounted directory) for 2nd, 3rd builds etc. also
- Checksum is providing better performance(> 50%) compared to the above for 2nd, 3rd builds etc.

cc @sanchitmehta

Reply:
Just FYI...the nightly builds also passed on these changes.

Reply:
Let's add a comment to this file with our findings regarding the timestamp usage and why we switched to checksums, so we don't go over this research again in the future

Reply:
Sure, will do